horizon/config.py

```diff
@@ -17,6 +17,36 @@ def __new__(cls, prefix=None, is_model=True):
             cls.instance = super(SidecarConfig, cls).__new__(cls)
         return cls.instance
 
+    ENABLE_GOPAL = confi.bool(
+        "ENABLE_GOPAL",
+        False,
+        description="if true, the sidecar will enable the GOPAL service to manage the PDP",
+    )
+
+    GOPAL_SERVICE_URL = confi.str(
+        "GOPAL_SERVICE_URL",
+        "http://localhost:8080",
+        description="URL to the Go opal service that manages this PDP, typically Permit.io cloud (api.permit.io)",
+    )
+
+    GOPAL_ENABLE_REMOTE_BACKUP = confi.bool(
+        "GOPAL_ENABLE_REMOTE_BACKUP",
+        False,
+        description="if true, the sidecar will enable the GOPAL service to manage the PDP",
+    )
+
+    GOPAL_REMOTE_BACKUP_URL = confi.str(
+        "GOPAL_REMOTE_BACKUP_URL",
+        None,
+        description="URL to the Go opal service that manages this PDP, typically Permit.io cloud (api.permit.io)",
+    )
+
+    GOPAL_TOKEN = confi.str(
+        "GOPAL_TOKEN",
+        "secret",
+        description="the token to authenticate with the GOPAL service",
+    )
+
```

Comment:
really don't like the gopal prefix, not sure what else we can do for now as the logic is split between python and go. see if you can come up with something better.

Reply:
changed